aedes.js

```diff
@@ -0,0 +1,118 @@
+'use strict'
+
+const WebSocket = require('ws')
+const tls = require('tls')
+const http = require('http')
+const net = require('net')
+const aedes = require('aedes')
+const { readFileSync } = require('fs')
+
+const DB = process.env.DB
+var persistence = 'aedes-persistence' + (DB ? '-' + DB : '')
+var mqemitter = 'mqemitter' + (DB ? '-' + DB : '')
+
+persistence = require(persistence)
+mqemitter = require(mqemitter)
+
+const servers = []
+
+const ports = {
+  TLS: 8883,
+  WS: 4000,
+  TCP: 1883
+}
+
+// first 2 args are node bin and aedes file
+// args will contain an array of server protocols
+// to start
+const args = process.argv.slice(2)
+
+const options = {
+  key: readFileSync('./server.key'),
+  cert: readFileSync('./server.cert'),
+  rejectUnauthorized: false
+}
+
+function listen (server, proto) {
+  return new Promise((resolve, reject) => {
+    server.listen(ports[proto], (err) => {
+      if (err) reject(err)
+      else {
+        console.log(proto, 'server is listening on port', ports[proto])
```

Comment:
I would remove all console.log. We might use aedes-logs if we have one, or rig something up with pino